sw/device/silicon_creator/rom/keys/fake/spx/dev_key_0_spx.pem

```diff
@@ -1,4 +1,4 @@
------BEGIN RAW SPHINCS+ PRIVATE KEY-----
+-----BEGIN RAW:SPHINCS+_SHA2_128s_simple+ PRIVATE KEY-----
```

Comment:
nit: It looks like there's an extra `+` in all of these. Is that necessary in some way for the `SphincsPlus::from_str` operation to work, or is it a typo?
```suggestion
-----BEGIN RAW:SPHINCS+_SHA2_128s_simple PRIVATE KEY-----
```

Reply:
Good catch.  I must have botched the `sed` expression to update the files.  Fixed.